lang-java/gradle/autobuild/build

```diff
@@ -1,17 +1,27 @@
+abinfo "Creating version.txt ..."
+echo $PKGVER > "$SRCDIR"/version.txt
+
+abinfo "Initializing the output dir ..."
+install -dv "$SRCDIR"/temp
+
+echo org.gradle.java.installations.fromEnv=JDK11,JDK17 >> gradle.properties
+
+abinfo "Building gradle ..."
+JDK11=/lib/java-11 JDK17=/lib/java-17 ./gradlew installAll -Pgradle_installPath="$SRCDIR"/temp -PfinalRelease=true --no-configuration-cache
```

Comment:
Why does it need two JDKs? How about using the minimal supported JDK version to build it.

Reply:
`--no-daemon`